sharpd/sharp_vty.c

```diff
@@ -27,6 +27,32 @@
 
 DEFINE_MTYPE_STATIC(SHARPD, SRV6_LOCATOR, "SRv6 Locator");
 
+DEFPY(watch_neighbor, watch_neighbor_cmd,
```

Comment:
Should we have a v6 watch too?

Reply:
This was a Proof of Concept bit of code.  I don't mind having it and I don't mind not having it.  An end operator is not likely to ever use this.